functions/Export-DbaLogin.ps1

```diff
@@ -268,7 +273,10 @@ function Export-DbaLogin {
 
                     $outsql += "`r`nUSE master`n"
                     # Getting some attributes
-                    $defaultDb = $sourceLogin.DefaultDatabase
+                    if ($DefaultDatabase) {
+                        $defaultDb = $DefaultDatabase } else {
```

Comment:
`$defaultDb = $DefaultDatabase } else {`

this is what's causing the failure. bracket and else should be on the next line. VS Code should have formatted that for you 🤔